src/com/company/CommandLineImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.company;
+
+public class CommandLineImpl implements CommandLine {
+    public CommandLineImpl() {
+    }
+
```

Comment:
Tutaj musimy miec wiecej interakcji z uzytkownikiem ktory odpala program, typu:
-> uzytkownik widzi przyciski: 
- pokaz liste produktow, 
- pokaz koszyk, 
- dodaj produkt do koszyka
- zloz zamowienie
- ...

-> wybiera liste produktow -> pokazuje sie lista produktow
-> to samo koszyk jak wyzej^

-> wybiera dodaj produkt -> dostaje napis: 'podaj id produktu', -> podaje id -> produkt dodaje sie do koszyka

to podstawowe scenariusze, mozna oczywiscie rozszerzyc